ledger_device_sdk/src/io.rs

```diff
@@ -331,6 +331,44 @@ impl Comm {
                 return None;
             }
 
+            // Default BOLOS APDU Handling
+            let apdu_header = self.get_apdu_metadata();
+            if apdu_header.cla == 0xB0 && apdu_header.p1 == 0x00 && apdu_header.p2 == 0x00 {
+                match apdu_header.ins {
+                    0x01 => {
```

Comment:
Can you put comments here to give the reader a hint of what the instructions are ? Or define a private enum (might be overkill...)